docs/data-cookbooks/sips-workflow.md

```diff
@@ -0,0 +1,121 @@
+  	        "regex": "^MOD09GQ\\.A[\\d]{7}\\.[\\S]{6}\\.006\\.[\\d]{13}\\.hdf$",
+  	        "sampleFileName": "MOD09GQ.A2017025.h21v00.006.2017034065104.hdf",
+  	        "url_path": "{cmrMetadata.Granule.Collection.ShortName}/{extractYear(cmrMetadata.Granule.Temporal.RangeDateTime.BeginningDateTime)}/{substring(file.name, 0, 3)}"
+  	    },
+  	    {
+  	        "bucket": "private",
+  	        "regex": "^MOD09GQ\\.A[\\d]{7}\\.[\\S]{6}\\.006\\.[\\d]{13}\\.hdf\\.met$",
+  	        "sampleFileName": "MOD09GQ.A2017025.h21v00.006.2017034065104.hdf.met"
+  	    },
+  	    {
+  	        "bucket": "protected-2",
+  	        "regex": "^MOD09GQ\\.A[\\d]{7}\\.[\\S]{6}\\.006\\.[\\d]{13}\\.cmr\\.xml$",
+  	        "sampleFileName": "MOD09GQ.A2017025.h21v00.006.2017034065104.cmr.xml"
+  	    },
+  	    {
+  	        "bucket": "public",
+  	        "regex": "^MOD09GQ\\.A[\\d]{7}\\.[\\S]{6}\\.006\\.[\\d]{13}_ndvi\\.jpg$",
+  	        "sampleFileName": "MOD09GQ.A2017025.h21v00.006.2017034065104_ndvi.jpg"
+  	    }
+    ],
+    "duplicateHandling": "replace",
+    "updatedAt": 1533313794693,
+    "url_path": "{cmrMetadata.Granule.Collection.ShortName}/{substring(file.name, 0, 3)}",
+    "timestamp": 1533313798525,
+    "stats": {
+        "running": 0,
+        "completed": 0,
+        "failed": 2,
+        "total": 2
+    }
+}
+```
+
+2. We need to have a [provider](./setup.md#providers) from whom data can be ingested. Our provider is an S3 provider hosted in the `cumulus-test-internal` bucket.
+
+![](../images/sips-provider.png)
+
+3. Finally, let's create a [rule](./setup.md#rules). In this example we're just going to create a `onetime` throw-away rule that will be easy to test with. This rule will kick off the `DiscoverAndParsePdrs` workflow, which is the beginning of a Cumulus SIPS workflow.
+
+```
+{
+    "name": "s3_provider_rule",
+    "workflow": "DiscoverAndQueuePdrs",
+    "provider": "s3_provider",
+    "collection": {
+        "name": "MOD09GQ",
+        "version": "006"
+    },
+    "rule": {
+        "type": "onetime"
+    },
+    "state": "ENABLED",
+    "tags": [
+        "test"
+    ]
+}
+```
+
+**Note:** A list of configured workflows exists under the "Workflows" in the navigation bar on the Cumulus dashboard. Additionally, one can find a list of executions and their respective status in the "Executions" tab in the navigation bar.
+
+
+## DiscoverAndQueuePdrs Workflow
+
+This workflow will (as the name might suggest) discover pdrs and queue them to be processed. Duplicate pdrs will be dealt with according to the configured duplicate handling setting.
+
+1. DiscoverPdrs - [npm package](https://www.npmjs.com/package/@cumulus/discover-pdrs), [source](https://github.com/nasa/cumulus/tree/master/tasks/discover-pdrs)
+2. QueuePdrs - [npm package](https://www.npmjs.com/package/@cumulus/queue-pdrs), [source](https://github.com/nasa/cumulus/tree/master/tasks/queue-pdrs)
+
+![](../images/sips-discover-and-queue-pdrs-execution.png)
```

Comment:
I really like the inclusion of images, but it seems like it'd be useful to give people something they could copy/paste/modify for their own workflows.  Could you add that?

Reply:
Great idea. I like @laurenfrederick 's suggestion to reference the source for the workflows in `example`. I've implemented this in my most recent changes.

Reply:
I have a slight concern about long-term readability / maintainability of that, but we can tackle that if it actually becomes a problem.  What I'd probably suggest is breaking up workflows.yml into one file per workflow, which would allow you to link directly to the external reference, rather than linking to a 700+ line file that people need to search through.  Out of scope here, though, and this addresses my concern.